### PR TITLE
Feat: Posting pictures and posts to the profile

### PR DESCRIPTION
**Pull request by lucamouchel**
This PR allows users to post to their profiles when they are on the art description activity. 
Now it looks like this:

### CODE REVIEW

**Review by hmiranda-queiros — CHANGES_REQUESTED** [review]
Good work ! However, you need to wait for the PR on Firebase Storage refactor to be merged before merging this one because you created functions that were already implemented in other PRs, but lost in the codebase because no class was created for Storage before like it is the case for the Database.
I also put other small questions/comments and maybe we need to discuss with the rest of the team to decide in which way to go.

File: app/src/main/java/ch/epfl/culturequest/database/Database.java
Comment (lucamouchel) [reply app/src/main/java/ch/epfl/culturequest/database/Database.java:260]: to me we should update each independently, especially bc many fields will never be updated (ie only the score, username and profile pic can be changed i think)

File: app/src/main/java/ch/epfl/culturequest/ui/profile/ProfileFragment.java
Comment (lucamouchel) [reply app/src/main/java/ch/epfl/culturequest/ui/profile/ProfileFragment.java:109]: right above, `limit = postsAdded`

**Review by hmiranda-queiros — CHANGES_REQUESTED** [review]
Good changes, I just put a comment on something you may need to change because of work being done in the scan fragment.

File: app/src/main/java/ch/epfl/culturequest/ui/profile/ProfileFragment.java
Comment (lucamouchel) [reply app/src/main/java/ch/epfl/culturequest/ui/profile/ProfileFragment.java:96]: just bc the function retrieve posts uses attributes limit and offset, but it is useless yes

**Review by hmiranda-queiros — APPROVED** [review]
All good to merge !

**Pull request merged**
